### PR TITLE
feat: add new strategies and countdown

### PR DESCRIPTION
**Pull request by joshuanatanielnm**
Related #646 #645 #642 #640 #638 

NOTE: This PR is mainly focus on the overview and landing page, gonna create other pr to make sure the detail page is run as expected

## Description

Add new strategies data, and make adjustment in hooks, interface, etc for new cellarType 

## Changes

List any technical changes.

- [x] add breakout eth and breakout btc in config
- [x] add new provider image
- [x] update strategy landing page content
- [x] create steadyBtc and steadyEth for cellarDataMap
- [x] Update overview page to have responsive row
- [x] [update hooks to allow PATACHE_LINK cellar key](https://github.com/strangelove-ventures/sommelier/commit/18b14aa2c330d702130d7e99d45f6e2866ebaeff)
- [x] [fix: get position to not return NaN if value is 0](https://github.com/strangelove-ventures/sommelier/commit/5af6fea6e9b6783b223a260ca209552cc57c3d90)\
- [x] [feat: update next config to prevent user go manage page](https://github.com/strangelove-ventures/sommelier/commit/9dbd9c088a16e80a8c0b9b5e9e74af9067a65176)
- [x] [feat: update chart to use performanceSplitKeys instaed of hardcoded](https://github.com/strangelove-ventures/sommelier/commit/7ae24b1a46cb8569bfdf30ccc43d72db964182bc) 
- [x] [feat: update hero right for coming soon](https://github.com/strangelove-ventures/sommelier/commit/7de177f4411dbeccfcd7da54202c733f48848d8f) 
- [x] [feat: update the about cellar for steady countdown](https://github.com/strangelove-ventures/sommelier/commit/afd0c14d60f113b5d988d031e471110c1c99444c)

## Screenshots (if appropriate):

image
image

### CODE REVIEW

File: src/components/HeroStrategy/hero-right.tsx [hunk 1]
```diff
@@ -38,85 +40,119 @@ export const HeroStrategyRight: VFC<HeroStrategyRightProps> = ({
...
+        <HStack
+          pt={4}
+          justifyContent="space-around"
+          alignItems="start"
+          divider={<StackDivider borderColor="purple.dark" />}
+        >
+          <VStack flex={1}>
+            <Heading size="md">{tokenPrice || "--"}</Heading>
+            <CellarStatsLabel
+              tooltip="The dollar value of the ETH, BTC, and USDC that 1 token can be redeemed for"
```
Comment (codingki) [inline src/components/HeroStrategy/hero-right.tsx:120]: this tool tip still using the momentum/trend, put this on `strategyPageContentData.ts`

File: src/components/HeroStrategy/hero-right.tsx [hunk 2]
```diff
@@ -38,85 +40,119 @@ export const HeroStrategyRight: VFC<HeroStrategyRightProps> = ({
...
-        divider={<StackDivider borderColor="purple.dark" />}
-      >
-        <VStack flex={1}>
-          <Heading size="md">{tokenPrice || "--"}</Heading>
-          <CellarStatsLabel
-            tooltip="The dollar value of the ETH, BTC, and USDC that 1 token can be redeemed for"
-            title="Token Price"
-          />
-        </VStack>
+      {countdown && (
```
Comment (codingki) [inline src/components/HeroStrategy/hero-right.tsx:57]: We can make it simple 
```
{
countdown ? (
 // countdown
)
: (
<>
// without countdown
</>
)
}

```

File: src/components/HeroStrategy/hero-right.tsx [hunk 3]
```diff
@@ -38,85 +40,119 @@ export const HeroStrategyRight: VFC<HeroStrategyRightProps> = ({
...
-              headingSize="md"
+          <VStack flex={1} textAlign="center">
+            {intervalGainPct.data ? (
+              <PercentageText
+                data={intervalGainPct.data}
+                headingSize="md"
+              />
+            ) : (
+              <Box>--</Box>
+            )}
```
Comment (codingki) [inline src/components/HeroStrategy/hero-right.tsx:141]: `PercentageText` component already handle null values so we don't need to make a ternerary options
```suggestion
            <PercentageText
                data={intervalGainPct.data}
                headingSize="md"
              />
```

File: src/components/HeroStrategy/hero-right.tsx [hunk 4]
```diff
@@ -38,85 +40,119 @@ export const HeroStrategyRight: VFC<HeroStrategyRightProps> = ({
...
+            <CellarStatsLabel
+              title={
+                isSteady
+                  ? "1W Change vs USDC"
+                  : "1W Change vs ETH/BTC 50/50"
```
Comment (codingki) [inline src/components/HeroStrategy/hero-right.tsx:146]: let's move "content" things in `uiConfig.ts` for example you can see `lpTokenTooltipContent` on `uiConfig.ts`. So we don't need to make a ternary options

**Review by codingki — CHANGES_REQUESTED** [review]

File: src/components/HeroStrategy/hero-right.tsx (hunk 1 above)
Comment (codingki) [reply src/components/HeroStrategy/hero-right.tsx:120]: let's move "content" things in uiConfig.ts for example you can see lpTokenTooltipContent on uiConfig.ts. So we don't need to make a ternary options

File: src/components/HeroStrategy/hero-right.tsx [hunk 5]
```diff
@@ -38,85 +43,104 @@ export const HeroStrategyRight: VFC<HeroStrategyRightProps> = ({
...
+          <HStack
+            pt={4}
+            justifyContent="space-around"
+            alignItems="start"
+            divider={<StackDivider borderColor="purple.dark" />}
+          >
+            <VStack flex={1}>
+              <Heading size="md">{tokenPrice || "--"}</Heading>
+              <CellarStatsLabel
+                tooltip="The dollar value of the ETH, BTC, and USDC that 1 token can be redeemed for"
```
Comment (codingki) [inline src/components/HeroStrategy/hero-right.tsx:116]: This is still using momentum and trend, add the content to `uiConfig.ts`

File: src/components/_cards/CellarCard/AboutCellar.tsx [hunk 6]
```diff
@@ -22,91 +24,107 @@ import { useDailyChange } from "data/hooks/useDailyChange"
...
+              />
+            )}
+          </Stack>
+
+          {isTokenPriceEnabled(cellarConfig) && (
             <CellarStats
-              tooltip="Total value managed by Strategy"
-              title="TVM"
-              value={tvm?.formatted || "..."}
+              tooltip="The dollar value of the ETH, BTC, and USDC that 1 token can be redeemed for"
```
Comment (codingki) [inline src/components/_cards/CellarCard/AboutCellar.tsx:76]: this is still using clear gate cellar content, branch the ui content in `uiConfig.ts`

File: src/components/_pages/PageCellar.tsx [hunk 7]
```diff
@@ -101,8 +102,14 @@ const PageCellar: VFC<CellarPageProps> = ({ id }) => {
                   arrow
                 />
               }
-              monthChangeTooltip="% change of token price compared to a benchmark portfolio of 50% ETH and 50% BTC"
-              monthChangeLabel="1W Change vs ETH/BTC 50/50"
+              monthChangeTooltip={`% change of token price compared to a benchmark portfolio of ${
+                isSteady ? "USDC" : "50% ETH and 50% BTC"
```
Comment (codingki) [inline src/components/_pages/PageCellar.tsx:106]: branch the content in `uiConfig.ts`

File: src/components/_pages/PageCellar.tsx [hunk 8]
```diff
@@ -101,8 +102,14 @@ const PageCellar: VFC<CellarPageProps> = ({ id }) => {
...
+              }`}
+              monthChangeLabel={
+                isSteady
```
Comment (codingki) [inline src/components/_pages/PageCellar.tsx:109]: branch the content in uiConfig.ts

File: src/data/strategies/steady-btc.ts [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+        </div>
+      </div>
+      Presented results are based on historical backtests. Past performance is not indicative of future results. Actual performance will depend on market conditions.`,
+  },
+  // overrideApy: {
+  //   title: "Backtested APY",
+  //   tooltip:
+  //     "Backtested APY results are based on historical backtests. Past performance is not indicative of future results. Actual performance will depend on market conditions",
+  //   value: "84.15%",
+  // },
```
Comment (codingki) [inline src/data/strategies/steady-btc.ts:75]: remove if not used

File: src/data/strategies/steady-eth.ts [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+        </div>
+      </div>
+      Presented results are based on historical backtests. Past performance is not indicative of future results. Actual performance will depend on market conditions.`,
+  },
+  // overrideApy: {
+  //   title: "Backtested APY",
+  //   tooltip:
+  //     "Backtested APY results are based on historical backtests. Past performance is not indicative of future results. Actual performance will depend on market conditions",
+  //   value: "84.15%",
+  // },
```
Comment (codingki) [inline src/data/strategies/steady-eth.ts:75]: remove if not used

**Review by xanderjl — APPROVED** [review]
LGTM 💪 Great work @joshuanatanielnm

File: next.config.js [hunk 11]
```diff
@@ -30,6 +30,19 @@ let nextConfig = {
...
+        destination: "/strategies/SteadyETH",
+        permanent: true,
+        basePath: false,
+      },
+      {
+        source: "/strategies/SteadyBTC/manage",
+        destination: "/strategies/SteadyBTC",
+        permanent: true,
+        basePath: false,
+      },
```
Comment (codingki) [inline next.config.js:45]: this looks like a hacky way, let's do it in `src/pages/strategies/[id]/manage.tsx` you need to filter the `id` that `launchDate` < today

**Review by codingki — CHANGES_REQUESTED** [review]

File: src/data/strategies/steady-btc.ts (hunk 9 above)
Comment (joshuanatanielnm) [reply src/data/strategies/steady-btc.ts:75]: I comment this out for the strategy details later

File: src/data/strategies/steady-eth.ts (hunk 10 above)
Comment (joshuanatanielnm) [reply src/data/strategies/steady-eth.ts:75]: I comment this out for the strategy details later

**Comment by joshuanatanielnm** [comment]
Thanks @codingki @xanderjl

**Pull request merged**